airbyte-scheduler/app/Dockerfile

```diff
@@ -5,9 +5,7 @@ ENV APPLICATION airbyte-scheduler
 
 WORKDIR /app
 
-COPY build/distributions/${APPLICATION}-0*.tar ${APPLICATION}.tar
-
-RUN tar xf ${APPLICATION}.tar --strip-components=1
+ADD build/distributions/${APPLICATION}-0*.tar /app
```

Comment:
Not related to your PR but why do we hav -0*. in the tar path. It will break when we will switch the major version to 1.

On another point could you ping me when this will get merge since I am working on those file at the moment.

Reply:
@benmoriceau PTAL. I'm thinking we can add this to each of the projects. If we add this to all of the projects, https://github.com/airbytehq/airbyte/issues/1605 will be resolved as well.

Reply:
With https://github.com/airbytehq/airbyte/pull/7542/files we should be good to close out #1605